faircmakemodules.sh

```diff
@@ -8,16 +8,16 @@ build_requires:
 ---
 #!/bin/sh
 
-cmake -S $SOURCEDIR -B .                                        \
-      -DCMAKE_INSTALL_PREFIX=$INSTALLROOT                       \
-      ${CMAKE_GENERATOR:+-G "$CMAKE_GENERATOR"}                 \
+cmake -S "$SOURCEDIR" -B .                                        \
+      -DCMAKE_INSTALL_PREFIX="$INSTALLROOT"                       \
+      ${CMAKE_GENERATOR:+-G "$CMAKE_GENERATOR"}                   \
       ${CMAKE_BUILD_TYPE:+-DCMAKE_BUILD_TYPE=$CMAKE_BUILD_TYPE}
 
-cmake --build . --target install
+cmake --build . --target install ${JOBS:+-- -j$JOBS}
 
 # Modulefile
 mkdir -p etc/modulefiles
-cat > etc/modulefiles/$PKGNAME <<EoF
+cat > etc/modulefiles/"$PKGNAME" <<EoF
```

Comment:
I think you can remove this completely and simply use alibuild-generate-module (see other places)